src/utils/content.ts

```diff
@@ -110,6 +110,25 @@ export const chapters: Chapter[] = [
     title: '05 - Basic Mutations',
     alias: 'rne-05',
   }]),
+  new Chapter('iOS', 'tutorial-ios', true, 'Learn how to easily get started iOS and GrahQL with Apollo Client. You will use the prepared application as a playground to experiment with an iOS Pokedex App.', [{
```

Comment:
change to reflect that the track is more than a playground. check other texts for comparison